README.md

```diff
@@ -78,6 +78,42 @@ def test_run_all_apply(run_all_apply_out):
     assert len(run_all_apply_out) == 3
 ```
 
+## Caching
+
+The `TerraformTest` `setup`, `init`, `plan`, `apply`, `output` and `destroy` methods have the ability to cache it's associate output to a local `.tftest-cache` directory. For subsequent calls of the method, the cached value can be returned instead of calling the actual underlying `terraform` command. Using the cache value can be significantly faster than running the Terraform command again especially if the command is time-intensive.
+
+To determine if the cache should be used, first a hash value is generated using the current `TerraformTest` instance `__init__` and calling method arguments. The hash value is compared to the hash value of the cached instance's associated arguments. If the hash is the same then the cache is used, otherwise the method is executed.
```

Comment:
This is awesome and makes it super clear why you've been pushing for this!

Reply:
Thank you, I hope it would be useful for others!